.travis.yml

```diff
@@ -248,7 +248,10 @@ script:
       cd "$BUILD_IN_DIR"
       travis_run_in_fold "build_in_dir" cmake $CMAKE_ARGS "$SOURCE_DIRECTORY"
     fi
-  - travis_run_in_fold "make" make
+  - travis_run_in_fold "make" make ; sleep 5
+  - make ; sleep 5
+  - make ; sleep 5
+  - make ; sleep 5
```

Comment:
@Elv13 
This ((1792780cf and 97a60818c)) shouldn't have been included here probably!?
Do you remember where it was coming from, and why the sleep might have been needed/useful?  (feel free to forward to Uli, of course :))

Reply:
For a while now, some Travis builds "just stop" in the middle of printing stuff. This was an attempt to get the logs. Like this:

https://api.travis-ci.com/v3/job/492704164/log.txt